migrations/20240331200101_initial.sql

```diff
@@ -2,7 +2,8 @@
 CREATE TABLE users (
     id INT PRIMARY KEY NOT NULL,
     token TEXT,
-    token_expiration BIGINT
+    token_expiration BIGINT,
+    check_timeout BIGINT
```

Comment:
do NOT edit existing sqlx migrations - this defeats the whole purpose of using sqlx which is to avoid database incompatibility and breakage after updates.

Reply:
please also recheck this